avm/res/network/p2s-vpn-gateway/tests/e2e/max/main.test.bicep

```
@@ -0,0 +1,80 @@
+targetScope = 'subscription'
+
+metadata name = 'Using only defaults'
+metadata description = 'This instance deploys the module with the minimum set of required parameters.'
```

Incorrect description. I guess you meant
```suggestion
metadata name = 'Using large parameter set'
metadata description = 'This instance deploys the module with most of its features enabled.'
```

---

Fixed; copy & paste error :-D